testing/visual-regression/backstop-common.js

```diff
@@ -32,6 +32,32 @@ module.exports = function backstopCommon(baseUrl) {
     ];
   }
 
+  function buildFormFieldScenarios(labelPrefix, componentId) {
+    const viewport = { label: 'form-field', width: 800, height: 400 };
+    return [
+      {
+        label: `${labelPrefix} (basic)`,
+        url: storyUrlFor(`${componentId}--basic`),
+        viewports: [viewport],
+      },
+      {
+        label: `${labelPrefix} (with hint)`,
+        url: storyUrlFor(`${componentId}--with-hint`),
+        viewports: [viewport],
+      },
+      {
+        label: `${labelPrefix} (optional field)`,
+        url: storyUrlFor(`${componentId}--optional-field`),
+        viewports: [viewport],
+      },
+      {
+        label: `${labelPrefix} (with error)`,
+        url: storyUrlFor(`${componentId}--with-error`),
+        viewports: [viewport],
```

Comment:
testing focus states would be good

Reply:
Agreed, might leave that to a follow on PR as the main goal of this one is to fix the display bug